frontend/src/components/home-events/index.tsx

```diff
@@ -126,30 +127,32 @@ export const Events = () => {
         </Column>
       </Row>
 
-      <Row
-        marginTop={{
-          desktop: -4,
-          tabletLandscape: -3,
-          tabletPortrait: 0,
-          mobile: 0,
-        }}
-        paddingLeft={STANDARD_ROW_PADDING}
-        paddingRight={STANDARD_ROW_PADDING}
-      >
-        <Column
-          columnWidth={{
-            mobile: 12,
-            tabletPortrait: 6,
-            tabletLandscape: 6,
-            desktop: 6,
+      <MaxWidthWrapper>
```

Comment:
Se a Container non metti fullWidth <Container fullWidth> non dovrebbe eservire questo wrapper

Reply:
I'll merge this for now, if you can improve it send a PR <3